pages/specification-links.md

```diff
@@ -399,24 +404,28 @@ _These were updated without changing functionality or meta-schemas due to a few
  - JSON Reference: [draft-pbryan-zyp-json-ref-03](https://tools.ietf.org/html/draft-pbryan-zyp-json-ref-03) ([changes](https://tools.ietf.org/html/draft-pbryan-zyp-json-ref-03#appendix-A))
  * [JSON Schema meta-schema](../../draft-04/schema)
  * [JSON Hyper-Schema meta-schema](../../draft-04/hyper-schema)
+ - Published: 2013
```

Comment:
```suggestion
 - Published: 31-January-2013
```

Reply:
@benjagm Thanks for reviewing
okk I'll fix this